### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by coldpigli**
1) Normal List
2) Lists with metadata
3) Numbered lists
Normal-lists
lists-meta-information
numbered-lists

### CODE REVIEW

**Review by nakulsharma15 — COMMENTED** [review]
Looks good to merge.

File: docs/components/lists.html [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+                        <a href="../way-forward-page.html">Way Forward</a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="side-nav-group gap-d20">
+            <h4 class="sub-title gap-d20">COMPONENTS</h4>
+            <div class="sub-list">
+                <ul>
+                    <li class="sub-list-items">
```
Comment (shubhamkumar648) [inline docs/components/lists.html:68]: as I see you used sub-list item class in every list tag  you can also use  one class  for list for this particular div

**Review by shubhamkumar648 — COMMENTED** [review]
consider my comment as a suggestion , reaming code  looks good

File: docs/components/lists.html (hunk 1 above)
Comment (coldpigli) [reply docs/components/lists.html:68]: Will look further into this. Thanks

**Pull request merged**